docs/pages/setup/reference/cli.mdx

```diff
@@ -311,6 +311,39 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy app
+
+Start local TLS proxy for app connections when using Teleport in single-port mode
+
+```code
+$ tsh proxy app [<flags>] <app>
+```
+
+#### Arguments
+`<app>`
+
+- `app` The name of the application to start the local proxy for.
```

Comment:
Should we specify that this needs to match the name of an application, e.g., as returned by `tsh apps ls`?

Reply:
Added a reference to this in https://github.com/gravitational/teleport/pull/10924/commits/2b9b9947be8a842d3f7deb415810eaebb1a14d75. Might need to be reworded but I think it makes it easier to understand.

Reply:
Looks great, thanks